packages/core/src/index.ts

```diff
@@ -27,6 +27,7 @@ export type {
   WalletConfigRekey,
   WalletExportImportConfig,
 } from './types'
+export * from './cache'
```

Comment:
The cache directory doesn't exist anymore in `main`. It's now in `packages/core/src/module/cache`. However the needed files should be exported already